automation/test.sh

```diff
@@ -30,6 +30,7 @@ export CDI_NAMESPACE=$NAMESPACE
 echo "namespace: ${NAMESPACE}, cdi-namespace: ${CDI_NAMESPACE}"
 
 readonly ARTIFACTS_PATH="exported-artifacts"
+readonly BAZEL_CACHE="${BAZEL_CACHE:-http://bazel-cache.kubevirt-prow.svc.cluster.local:8080/kubevirt.io/kubevirt}"
```

Comment:
the path of the url should be `kubevirt.io/containerized-data-importer`.

Reply:
Fixed